src/systems/userinput/bindings/vive-user.js

```diff
@@ -315,21 +348,31 @@ export const viveUserBindings = addSetsToBindings({
     {
       src: {
         bool: rTouchpadRising,
-        value: rDpadEast
+        value: rDpadEast1
       },
       dest: {
-        value: rSnapRight
+        value: rSnapRight1
       },
       xform: xforms.copyIfTrue,
       priority: 1
     },
+    {
+      src: {
+        value: rDpadEast2
+      },
+      dest: {
+        value: rSnapRight2
+      },
+      xform: xforms.rising,
+      priority: 1
```

Comment:
I think this priority value is unnecessary

Reply:
Ah yes, forgot about that one, thanks.